Gemfile

```diff
@@ -80,20 +81,21 @@ group :test do
   gem 'database_cleaner', '~> 1.6.1'
   gem 'email_spec', '~> 2.1.0'
   gem 'rspec-rails', '~> 3.6'
-  gem 'selenium-webdriver', '~> 3.10'
+  gem 'selenium-webdriver', '~> 3.11.0', require: false
 end
 
 group :development do
   gem 'capistrano', '~> 3.10.1', require: false
+  gem 'capistrano3-delayed-job', '~> 1.7.3'
   gem 'capistrano-bundler', '~> 1.2', require: false
   gem 'capistrano-rails', '~> 1.3.1', require: false
-  gem 'capistrano3-delayed-job', '~> 1.7.3'
+  gem 'fuubar', '~> 2.3.1'
```

Comment:
While working on #1611 isn't in the `Gemfile` anymore in the `master` branch. Does it make sense to remove this line from this pull request?

Reply:
It's not even on Consul's `master` branch :sweat_smile: and I don't see it being used anywhere so let's remove it for the time being :ok_hand: